src/vmm/tests/integration_tests.rs

```diff
@@ -305,7 +305,7 @@ fn verify_create_snapshot(is_diff: bool) -> (TempFile, TempFile) {
 
             // Create snapshot.
             let snapshot_type = match is_diff {
-                true => SnapshotType::Diff,
+                true => unimplemented!(),
```

Comment:
nit: As `bool` has only two possible values, it would be nice to use `true/fasle` instead of `_` when pattern matching.

Reply:
done